README.md

```diff
@@ -136,6 +136,10 @@ If you want to work on your dependencies during development, and want them to be
 
 Note that you should be [using submodules](#using-submodules-for-dependencies) before doing this, because plain checkouts [should not be modified][Carthage/Checkouts] directly.
 
+### Caching builds
+
+By default Carthage will rebuild a dependency regardless of whether it's the same resolved version as before. Passing the `--cache-builds` will cause carthage to avoid rebuilding a dependency if it can. See information on [version files][VersionFile] for details on how carthage performs this caching. 
```

Comment:
This isn't going to be by default for now. This needs to be updated to reflect that it's off by default.